src/php/lib/Grpc/BaseStub.php

```diff
@@ -19,6 +19,77 @@
 
 namespace Grpc;
 
+abstract class Interceptor
+{
+  public function __construct(){}
+}
+
+class ClientInterceptor extends Interceptor
```

Comment:
Didn't find a better way to let user just implement parts of the method with the interface exactly as the parent method.

I was trying to use Interface/abstractClass, but that requires users implement all 4 methods even if the user just want the UnaryCall interceptor.